iam/api-client/src/test/java/iam/snippets/ServiceAccountTests.java

```diff
@@ -109,21 +112,21 @@ public void stage2_testServiceAccountKeysList() {
   @Test
   public void stage2_testServiceAccountKeyDisable() {
     DisableServiceAccountKey
-        .disableServiceAccountKey(PROJECT_ID, SERVICE_ACCOUNT, SERVICE_ACCOUNT_KEY);
+            .disableServiceAccountKey(PROJECT_ID, SERVICE_ACCOUNT, SERVICE_ACCOUNT_KEY);
     String got = bout.toString();
     assertThat(got, containsString("Disabled service account key"));
   }
 
   @Test
   public void stage2_testServiceAccountKeyEnable() {
     EnableServiceAccountKey
-        .enableServiceAccountKey(PROJECT_ID, SERVICE_ACCOUNT, SERVICE_ACCOUNT_KEY);
+            .enableServiceAccountKey(PROJECT_ID, SERVICE_ACCOUNT, SERVICE_ACCOUNT_KEY);
     String got = bout.toString();
     assertThat(got, containsString("Enabled service account key"));
   }
 
   @Test
-  public void stage3_testServiceAccountKeyDelete() {
+  public void stage3_testServiceAccountKeyDelete() throws IOException {
     DeleteServiceAccountKey.deleteKey(PROJECT_ID, SERVICE_ACCOUNT, SERVICE_ACCOUNT_KEY);
     String got = bout.toString();
     assertThat(got, containsString("Deleted key:"));
```

Comment:
Consider retrieving the service account to confirm that the key has been deleted.